test/ProductConstructionService.Api.Tests/JobsProcessorStatusTests.cs

```diff
@@ -2,27 +2,39 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using ProductConstructionService.Api.Metrics;
 using ProductConstructionService.Api.Queue;
 
 namespace ProductConstructionService.Api.Tests;
 
 public class JobsProcessorStatusTests
 {
+    private readonly IServiceProvider _serviceProvider;
+
+    public JobsProcessorStatusTests()
+    {
+        ServiceCollection services = new();
+
+        services.AddSingleton(new Mock<IMetricRecorder>().Object);
```

Comment:
Should we not verify that the metric for job duration was called?

Reply:
It won't happen in this test, since we're not actually running any jobs, just creating scopes. I'll write a test for this